tensorflow/core/grappler/optimizers/meta_optimizer.cc

```diff
@@ -93,6 +98,10 @@ Status MetaOptimizer::Optimize(Cluster* cluster, const GrapplerItem& item,
       optimizers.push_back(std::unique_ptr<GraphOptimizer>(
           new ArithmeticOptimizer(cfg_.arithmetic_optimization())));
     }
+    if (cfg_.loop_optimization() != RewriterConfig::OFF) {
```

Comment:
Can you change this to   if (cfg_.loop_optimization() == RewriterConfig::ON) for now ? We typically find a few issues whenever we add a new optimization, so we always start by turning new optimizers OFF by default, put the code through its paces (we have a lot of models to try this on internally), solve the problems we uncover, and eventually turn it ON by default.

Reply:
Thanks for the info. I have made this change in the new commit.